docs/Calendar/README.md

```diff
@@ -61,7 +61,7 @@ class ExampleBpkCalendarController(
   private val context: Context,
   override val selectionType: SelectionType
  ) : BpkCalendarController(selectionType) {
-  override fun onRangeSelected(range: CalendarSelection) {
+  override fun onRangeSelected(selection: CalendarSelection) {
```

Comment:
I know nothing about Android, but I can only find usages of `onRangeSelected(range:...)`. Are you sure it's meant to say `selection`?

Reply:
It's the variable name for the function. The body uses `selection` but it was named `range` leading to compilation error on copying from docs.

Reply:
Maybe we should rename the selection references to range instead, so it matches with the API